x-pack/plugins/alerting/server/authorization/alerting_authorization.ts

```diff
@@ -302,8 +306,11 @@ export class AlertingAuthorization {
       );
 
       const authorizedEntries: Map<string, Set<string>> = new Map();
+      const alertsSpaceId =
```

Comment:
I would probably prefer to see this passed into the `getFindAuthorizationFilter` instead of having this function become entity aware.

Reply:
Yea, I had gone back and forth about it. I agree with you. I went back to making it an option for `getFindAuthorizationFilter`.